src/pages/detailedNotice/ZaboShowcase.tsx

```diff
@@ -79,20 +79,24 @@ const ZaboShowcase = ({ src, onShow }: ZaboShowcaseProps) => {
             onClick={onShow}
           />
         </div>
-        <ImageCTAButton
-          variant={ButtonVariant.contained}
-          height={"40px"}
-          left={ctaButtonLeft}
-          ref={ctaButtonRef}
-          onClick={onShow}
-        >
-          <Flex gap={"16px"}>
-            <Text size={"0.875rem"} font={Font.Medium}>
-              포스터 클릭하여 자세히 보기
-            </Text>
-            <Icon.ArrowDownWhite width={"16px"} />
-          </Flex>
-        </ImageCTAButton>
+        {onShow && (
+          <ImageCTAButton
+            variant={ButtonVariant.contained}
+            height={"40px"}
+            left={ctaButtonLeft}
+            ref={ctaButtonRef}
+            onClick={() => {
+              onShow?.();
+            }}
```

Comment:
여기 밑으로 내려오면 onShow는 nullable 아니지 않나요?

```suggestion
            onClick={onShow}
```